tests/activity_test.go

```diff
@@ -271,27 +271,42 @@ func (s *ActivityClientTestSuite) Test_ActivityTimeouts() {
 		f3 := workflow.ExecuteActivity(ctx3, activityFn)
 
 		ctx4 := workflow.WithActivityOptions(ctx, workflow.ActivityOptions{
+			ActivityID:          "ScheduleToCloseNotSet",
```

Comment:
Have you considered test cases and sub tests approach? It makes tests more verbose.